WebContent/EditParty.jsp

```diff
@@ -0,0 +1,88 @@
+
+<%@page import="model.Party"%>
+ <%@page import="dao.SearchByPartyId"%> 
+
+<html>
+<head>
+<title>Party Management App</title>
+<link rel="stylesheet"
+	href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+	integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+	crossorigin="anonymous">
+</head>
+<body>
+
+	<header>
+		<nav class="navbar navbar-expand-md navbar-dark"
+			style="background-color: tomato">
+			<div>
+				<a href="#" class="navbar-brand"> Party Management App </a>
+			</div>
+
+			<ul class="navbar-nav">
+				<li><a href="<%=request.getContextPath()%>index.jsp"
+					class="nav-link">Party</a></li>
+			</ul>
+		</nav>
+	</header>
+	<br>
+	<div class="container col-md-5">
+		<div class="card">
+			<div class="card-body">
+
+
+				<form action="UpdateServlet" method="Post">
+
+				<%
+					int id = Integer.parseInt(request.getParameter("id"));
+							SearchByPartyId ud = new SearchByPartyId();
+							
+							Party  p = ud.searchById(id);
```

Comment:
Please use proper variable names p, id and ud are the bad examples of variable names.